crates/subspace-proof-of-space/src/chiapos/table.rs

```diff
@@ -119,7 +119,7 @@ fn partial_ys<const K: u8>(seed: Seed) -> Vec<u8> {
 
     let mut cipher = ChaCha8::new(&key, &iv);
 
-    cipher.write_keystream(&mut output);
+    cipher.apply_keystream(&mut output);
```

Comment:
This is the most suspicious change, maybe this is the wrong function name or function arguments?

Reply:
This is the right function and right arguments. Feel free to revert and try again though, but I'm quite confident it is correct. The argument type has changed because of the version of the library.